lib/main.js

```diff
@@ -274,6 +288,16 @@ proto.rating = {
     if (el && !this.rating._isElSelected(el)) {
       el.click();
     }
+  },
+
+  // Reset the rating
+  resetRating: function () {
+    var selector = SELECTORS.rating.thumbSelectorFormat.replace('{rating}', this.rating.getRating());
+    var el = this.doc.querySelector(selector);
+
+    if (el) {
```

Comment:
Logically if we just fetched the currently selected rating toggle it should still be selected 2 lines later but again I'll add it for sake of clarity